beater/config.go

```diff
@@ -1,15 +1,15 @@
 // Config is put into a different package to prevent cyclic imports in case
 // it is needed in several locations
 
-package config
+package beater
 
 import "github.com/elastic/beats/libbeat/common"
 
-type Config struct {
-	Host   string         `config:"host"`
-	Server *common.Config `config:"server"`
+type BeaterConfig struct {
```

Comment:
Now it's `beater.BeaterConfig`. +1 on moving the package but the name should stay just `Config` because then it becomes `beater.Config`.